### PR TITLE
Document zipkin http endpoint

### PR DESCRIPTION
**Pull request by pavolloffay**

### CODE REVIEW

File: docs/deployment.md [hunk 1]
```diff
@@ -56,7 +56,8 @@ At default settings the collector exposes the following ports:
 Port  | Protocol | Function
 ----- | -------  | ---
 14267 | TChannel | used by **jaeger-agent** to send spans in jaeger.thrift format
-14268 | HTTP     | can accept spans directly from clients in Jaeger or Zipkin Thrift 
+14268 | HTTP     | can accept spans directly from clients in jaeger.thrift format
+9411  | HTTP     | can accept Zipkin spans in JSON or Thrift (disabled by default)
```
Comment (black-adder) [inline docs/deployment.md]: do we want to be more precise? ie hostport:/api/v1/spans?

Comment (pavolloffay) [inline docs/deployment.md]: I think this is fine it uses the same port so users assume that it is the same API and it's mostly for users migrating from zipkin (I guess)

Comment (pavolloffay) [inline docs/deployment.md]: There is also this http://jaeger.readthedocs.io/en/latest/getting_started/#migrating-from-zipkin. We should probably say there that we have zipkin compatible REST API.

Comment (black-adder) [inline docs/deployment.md]: +1

**Pull request merged**